mobile_config/src/mobile_radio_tracker.rs

```diff
@@ -86,17 +88,23 @@ impl TrackedMobileRadio {
             hash: radio.hash(),
             last_changed_at: radio.refreshed_at,
             last_checked_at: Utc::now(),
+            asserted_location: radio.location,
+            asserted_location_change_at: None,
```

Comment:
What is the thinking about making this None?  The first time we see a radio, this field will be null and how do you envision clients working with it?